test/augmentation/test_functional_torchvision.py

```diff
@@ -0,0 +1,130 @@
+"""
+Temporary devloping file to test kornia augmentation functions aginst torchvision.
+
+Note:
+1. In torchvision, the conversion from (0, 1) to uint8 then to (0, 1) has info losses.
+"""
+from typing import Tuple
+
+import pytest
+import torch
+import torch.nn as nn
+import math
+
+from torch.testing import assert_allclose
+from torch.autograd import gradcheck
+
+import kornia
+import kornia.testing as utils  # test utils
+import kornia.augmentation.functional as F
+import kornia.augmentation.param_gen as pg
+# TODO: Should we have a class to store those constants?
```

Comment:
Why a class? Actually, was thinking the other day that pi is now a tensor, but at least for last versions in Pytorch just declaring it as float might work. Any thoughts about it? /cc @ducha-aiki

Reply:
I prefer floats and constants, not classes :)

Reply:
Sorry, I mean to have a separate file like kornia.constant to store the constant variables. To me, it is a bit weird to import PI from the geometry module.

Reply:
Sure. We could have in the root something like kornia._constants.py